src/talos/DownloadPanelApp.py

```diff
@@ -0,0 +1,417 @@
+"""
+Prior to the creation of this script, PanelApp was queried on two separate occasions per run:
+
+- once to get all the panels in PanelApp, and the associated HPO terms for each
+- once to get the gene data for each panel
+
+There's a couple of ways in which this is wasteful:
+- PanelApp is only updated once per month, but this querying goes on in every run
+- the original API query implementation was in series, not parallel, due to server instability, so it was slow
+
+In addition to the above, the core assumption here is that deployment environments will have an internet connection.
+That is typically not true of HPC environments, making this a tough workflow to run in those environments.
+
+This script queries for the whole of PanelApp in one go, using a barrage of highly parallelised queries:
```

Comment:
We need to discuss this with Simon. Fine while it is a couple of users, but he might want to provide an endpoint for a nightly pre-rendered file for direct download or similar to avoid the load.

Reply:
Yup, that makes sense. This implementation is the same overall amount of querying as a single run, but it's way more highly parallelised so the point burden could be much harder. So far so good, but I'm the only user 😅